listenbrainz/db/model/utils.py

```diff
@@ -0,0 +1,42 @@
+import uuid
```

Comment:
I see this file has already been merged in #1503. However, I think `validators.py` is a better name for this file. Can you please rename it?